stratum/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: v1
+apiVersion: v2
 name: stratum
 version: 0.1.4
```

Comment:
Shouldn't we bump the Chart version?
It looks like [helmrepo.sh](https://gerrit.opencord.org/plugins/gitiles/helm-repo-tools/+/refs/heads/master/https://gerrit.opencord.org/helm-repo-tools) enforce the chart version change, similar to what onos-helm-chart does. Is that correct?

Reply:
This version is used for Helm tool version, v1 for Helm 2, and v2 for Helm3. 
Because the Rancher already supports the Helm3, I suggest upgrading to the Helm3.